File: algorithms/283.Move_Zeroes/Move_Zeroes.cc
<<<<<<< HEAD
class Solution {
public:
    void moveZeroes(vector<int>& nums) {
        int len = 0;
        for (int i = 0; i < nums.size(); ++i) {
            if (nums[i] != 0) {
                if (nums[len] != nums[i]) {
                    swap(nums[len], nums[i]);
                }
                ++len;
            }
        }
    }
=======
class Solution {
public:
    void moveZeroes(vector<int>& nums) {
        int len = 0;
        for (int i = 0; i < nums.size(); ++i) {
            if (nums[i] != 0) {
                if (nums[len] != nums[i]) {
                    swap(nums[len], nums[i]);
                }
                ++len;
            }
        }
    }
>>>>>>> 6200c8704614e918c8bfa5357c648dd1b4f7eb74
};

File: algorithms/283.Move_Zeroes/Move_Zeroes.py
<<<<<<< HEAD
class Solution(object):
    def moveZeroes(self, nums):
        """
        :type nums: List[int]
        :rtype: void Do not return anything, modify nums in-place instead.
        """
        nums.sort(cmp= lambda a, b: -1 if b == 0 else 0)
=======
class Solution(object):
    def moveZeroes(self, nums):
        """
        :type nums: List[int]
        :rtype: void Do not return anything, modify nums in-place instead.
        """
        nums.sort(cmp= lambda a, b: -1 if b == 0 else 0)
>>>>>>> 6200c8704614e918c8bfa5357c648dd1b4f7eb74
